plenum/test/input_validation/constants.py

```diff
@@ -0,0 +1,16 @@
+import base58
+
+VALID_BASE58 = base58.alphabet
+
+TEST_B58_BY_DECODED_LEN = {
+    len(base58.b58decode(VALID_BASE58[:i])): VALID_BASE58[:i] \
+        for i in range(1, len(VALID_BASE58) + 1)
+}
```

Comment:
Lets make `TEST_B58_BY_DECODED_LEN` much explicit because it is not simple to get what actually the variable contains.